.github/workflows/conventional-pr-title-checker.yml

```diff
@@ -7,20 +7,14 @@ on:
       - reopened
       - edited
       - synchronize
-  push:
-    branches:
-      - gh-readonly-queue/main/**
```

Comment:
Why are removing that? Can't we keep it in case we want to enable merge queues at some point? Took me a bunch of iterations to figure this one out, would be nice to not go trhough them again

Reply:
On the contrary one could look up the git history like I just did?

Reply:
But they would need to know that some changes are required on this file first. Unless they are ok browsing the entire git history, which is very unlikely.
What's our stand with merge queue? Do we want to push it and try to make it work with release please? In which case I'd like to keep that around. If not, then yeah let's remove it

Reply:
I have reverted the others and just have the title checker file changed as it uses `pull_request_target` instead of `pull_request` which makes `github.ref` not unique for it.

Reply:
If we switch it to just `pull_request` then a forked PR will not have write permissions which feels alright to me. Let me try that out.

Reply:
The check writes a status, it seems like the check with `pull_request` passes/fails without the status permissions and with `pull_request` this might just work.

Reply:
After this gets merged I'll try to create a PR from a fork and see if that works to verify behavior for outsiders.